x/config/config.go

```diff
@@ -93,6 +94,14 @@ func newStreamDialerFromPart(innerDialer transport.StreamDialer, oneDialerConfig
 	case "tls":
 		return newTlsStreamDialerFromURL(innerDialer, url)
 
+	case "tlsfrag":
```

Comment:
We should align with the tcp fragmentation. I don't think it makes sense to have both.
Should we rename `split` to `frag`? We'll need to update the documentation.
We could also make `frag` be a synonym for `split` and deprecate `split`.

Reply:
I can rename `split` to `frag` since [this document](https://upb-syssec.github.io/blog/2023/record-fragmentation/#tcp-fragmentation) also uses the term `TCP Fragmentation`.

Reply:
Hmm, in this case I would prefer `split` cuz `fragmentation` is too long.

Reply:
It can be `fragment` as well, as a verb